<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>768</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Source Code Pro Semibold</family>
    <pointsize>11</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Batalla Naval v0.17.01</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-games" resource="src.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
	border-image: url(:/images/bg.png);
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>768</height>
    </size>
   </property>
   <widget class="QPushButton" name="cargarPartida">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>316</y>
      <width>440</width>
      <height>90</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>480</width>
      <height>110</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>25</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#cargarPartida {
  background-color: #ddd;
  border: 10px inset #5fcfbf;
  color: black;
  padding: 10px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  margin: 4px 2px;
  cursor: pointer;
  border-radius: 40px;
}

#cargarPartida:hover {
  background-color: #188c8c;
}

</string>
    </property>
    <property name="text">
     <string>Cargar Partida</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nuevoJuego">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>440</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>480</width>
      <height>110</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>25</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#nuevoJuego {
  background-color: #ddd;
  border: 10px inset #5fcfbf;
  color: black;
  padding: 10px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  margin: 4px 2px;
  cursor: pointer;
  border-radius: 40px;
}

#nuevoJuego:hover {
  background-color: #188c8c;
}

</string>
    </property>
    <property name="text">
     <string>Nuevo Juego</string>
    </property>
   </widget>
   <widget class="QWidget" name="configJuego" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QPushButton" name="jugar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>JUGAR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tamanioMapa">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="cantBarcos">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>35</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkAleatorio">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>25</y>
       <width>211</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>AÃ±adir Aleatorio</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mapas" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>1231</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QWidget" name="mapaJugador" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>661</width>
       <height>291</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>501</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="mapGLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="mapaAtaque" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>661</width>
       <height>281</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>501</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="attackGLayout"/>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="listaAtaquesJugador">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>350</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="infoBarcos">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>40</y>
       <width>471</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="listaAtaquesIA">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>390</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="guardarPartida">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>450</y>
       <width>471</width>
       <height>90</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>480</width>
       <height>110</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
       <pointsize>25</pointsize>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#guardarPartida {
  background-color: #ddd;
  border: 10px inset #5fcfbf;
  color: black;
  padding: 10px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  margin: 4px 2px;
  cursor: pointer;
  border-radius: 40px;
}

#nuevoJuego:hover {
  background-color: #188c8c;
}

</string>
     </property>
     <property name="text">
      <string>Guardar Partida</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ponerBarcos" native="true">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>440</y>
      <width>381</width>
      <height>160</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <widget class="QPushButton" name="aniadirBarco">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>100</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>AÃADIR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="posY">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QComboBox" name="alineacion">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="currentText">
      <string>V</string>
     </property>
     <item>
      <property name="text">
       <string>V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>H</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="posX">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QComboBox" name="listaBarcos">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A VER QUE VERGA ES ESTO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>CRUCERO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DESTRUCTOR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FRAGATA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LANCHA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PORTAAVION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SUBMARINO</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="barcosRestantes">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>64</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="previewBarco" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>491</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>441</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="preview">
      <item>
       <widget class="QLabel" name="imagenBarco">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nombreBarco">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>configJuego</zorder>
   <zorder>cargarPartida</zorder>
   <zorder>mapas</zorder>
   <zorder>ponerBarcos</zorder>
   <zorder>previewBarco</zorder>
   <zorder>nuevoJuego</zorder>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
